ext/ffi/ffi_parser.c

```diff
@@ -313,7 +313,7 @@ static int get_skip_sym(void) {
 	int ch;
 	int ret;
 	int accept = -1;
-	const unsigned char *accept_pos;
+	const unsigned char *accept_pos = NULL;
```

Comment:
This change is moot, since ffi_parser.c is generated from ffi.g, so the change would likely have to be made at https://github.com/dstogov/llk/blob/70b057f49948a1f5bb37cc5cb1a324605c3903e3/llk_c.php#L262.